### PR TITLE
3-Forgot-Password-Page (resolves #3)

### LINKED ISSUE #3: Create "Forgot Password" Page
### Description
Create a basic forgot password page based on [Figma](https://www.figma.com/file/MNRQAXLXzBqCR3aRZmXg0T/Habitat-for-Humanity). Include error handling for input fields. If an error occurs, an alert pops up describing the error. Store all input fields in a state variable so it can be formatted as a JSON object later

### Setup
1. Navigate to `/frontend/src/components/`
2. Create a folder with the [PascalCase](https://techterms.com/definition/pascalcase) naming convention (name based on the component you're working on)
3. Inside the folder, create a `.tsx` file with the same name as the folder
4. Be sure to update the `App.js` file with your new component, or else you wont see any changes you make. This file is located in `/frontend/src/`

### Success Metrics Checklist
- [x] Looks exactly like the Figma
- [x] Responsive screen handling
- [x] Error pops up for invalid/empty input field
- [x] Store submitted input into state variable

Finally, on your pull request, please request a code review from me (@ObjectJosh)

Screen Shot 2022-01-06 at 1 09 16 PM

### PR DESCRIPTION
**Pull request by ObjectJosh**

### CODE REVIEW

File: frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    border: 3px solid rgba(2, 5, 12, 0.15)
+}
+
+.inputBox {
+    font-family: Rubik;
+    box-sizing: border-box;
+    height: 43px;
+    width: 100%;
+    border: 1px solid var(--input-box);
+    padding: 0px;
```
Comment (ObjectJosh) [inline frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css:78]:
```suggestion
    padding: 9px;
```
Just adding some padding:
Screen Shot 2022-01-17 at 7 27 57 PM

File: frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+.inputBox {
+    font-family: Rubik;
+    box-sizing: border-box;
+    height: 43px;
+    width: 100%;
+    border: 1px solid var(--input-box);
+    padding: 9px;
+    font-size: 13px;
+    font-weight: 500;
+}
```
Comment (tessa-tapmongkol) [inline frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css:81]: Add css for smaller view ports. You can do this by using `@media only screen and (max-width: ) { }` or  `@media only screen and (max-height: ) { }` 
For example, to change the width of #forgotPasswordBox at up to 500px you would do:
`@media only screen and (max-width: 500px) {
    #forgotPasswordBox {
        width: 200px
    }
}`
And you can use this for even smaller viewports.

File: frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+#forgotPasswordText {
+    font-family: Rubik;
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 40px;
+    margin: 0px 0px 34px 0px;
+    text-align: left;
+
+    color: var(--black);
```
Comment (tessa-tapmongkol) [inline frontend/src/components/ForgotPasswordPage/ForgotPasswordPage.css:29]: nit: the color on Figma is an off black (#011338) not sure how much this matters

**Review by tessa-tapmongkol — CHANGES_REQUESTED** [review]

**Comment by ObjectJosh** [comment]
3-Forgot-Password-Page-Fixes

**Pull request closed without merging**